benchmarks/runner/benchmarks/append.js

```diff
@@ -9,7 +9,7 @@ const base = {
     return { log, ipfs, repo }
   },
   cycle: async function ({ log }) {
-    await log.append('Hello')
+    await log.append('Hello', 32)
```

Comment:
Why 32 here?

Reply:
In the `orbit-db-store` PR I changed the default `referenceCount` to 32, so I changed it here to make the benchmark more accurately reflect that.

Reply:
:+1: